galaxy_ng/app/migrations/0030_update_collection_remote_rhcertified_url.py

```diff
@@ -0,0 +1,29 @@
+from django.db import migrations
+
+def update_collection_remote_rhcertified_url(apps, schema_editor):
+    """
+    Updates the existing collection remote `rh-certified` url field  
+    if startswith `https://cloud.redhat.com/`.
+    """
+    
+    CollectionRemote = apps.get_model('ansible', 'CollectionRemote')
+    
+    rh_remote = CollectionRemote.objects.get(name='rh-certified')
```

Comment:
The `get()` method will raise an exception if for some reason the rh-certified collection does not exist, which we probably don't want in a migration. This change will make it return `None` if it does not exist.

```suggestion
    rh_remote = CollectionRemote.objects.filter(name='rh-certified').first()
```

Reply:
makes sense, fixed :slightly_smiling_face: